src/utils/helper.js

```diff
@@ -0,0 +1,25 @@
+export function setButtonText(
+  btn,
+  isLoading,
+  defaultText = "Save",
+  loadingText = "Saving..."
+) {
+  if (isLoading) {
+    btn.textContent = loadingText;
+  } else {
+    btn.textContent = defaultText;
+  }
+}
+
+export function deleteButtonText(
```

Comment:
Function names should start with a **verb** in infinitive (there is such a point in the checklist). It’s common in coding to do it so that functions could `speak` for themselves about what they do. **Please, correct it everywhere in the code**


```js

export function setDeleteButtonText(

```